src/manufacturer/manufacturer.component.tsx

```diff
@@ -60,6 +65,9 @@ function ManufacturerComponent() {
         Header: TableHeaderOverflowTip,
         accessorFn: (row) => row.name,
         id: 'name',
+        filterVariant: COLUMN_FILTER_VARIANTS.string,
+        filterFn: COLUMN_FILTER_FUNCTIONS.string,
+        columnFilterModeOptions: COLUMN_FILTER_MODE_OPTIONS.string,
         size: 400,
         Cell: ({ row }) =>
           row.original.name && (
```

Comment:
```suggestion
        Header: TableHeaderOverflowTip,
        accessorFn: (row) => row.name,
        id: 'name',
        filterVariant: COLUMN_FILTER_VARIANTS.string,
        filterFn: COLUMN_FILTER_FUNCTIONS.string,
        columnFilterModeOptions: COLUMN_FILTER_MODE_OPTIONS.string,
        size: 400,
        Cell: ({ row }) =>
         (
```
name is a mandatory field so the conditional operator is not needed